internal/pipeline/executor.go

```diff
@@ -190,8 +190,8 @@ func (p *pkg) localResources(includeMetadata bool) (resources []*yaml.RNode, err
 		err = fmt.Errorf("failed to read resources for pkg %s %w", p.Path(), err)
 		return resources, err
 	}
+	resources = filterFnConfig(resources)
 	if !includeMetadata {
```

Comment:
Not sure about here. Maybe @droot can clarify. Should we put `filterFnConfig` in this if?

Reply:
moved the isfnconfig check to inside the filtermetaresources function. @droot wdyt?